src/driver/include/driver/DefaultDriver.hpp

```diff
@@ -64,6 +64,7 @@ class DefaultDriver {
         std::string metricsOutputFileName;
         std::string mongoUri;
         std::string description;
+        bool isSmokeTest;
```

Comment:
I decided not to make smoke test a run mode so people can use `evaluate` to see what the smoke test does.

It's possible for the smoke test to still take a lot of time if a single iteration of a phase takes a long time to run; so people may still want to manually modify their workload configs to run the smoke test.

Reply:
> I decided not to make smoke test a run mode so people can use evaluate to see what the smoke test does.

Should it instead be a flag to `genny evaluate`? E.g. `genny evaluate --smoke-test`?

That seems reasonable but I think we'd want to change the current dry-run CI task to actually do `smoke-test` (or perhaps keep it and add an additional task). The one gotcha would be that workload YAMLs would need to specify their topology requirements but that's probably a good thing to have anyway. 

So it's fine to leave it off as a "run mode" but hopefully it remains a simple shell operation to `genny dry-run > workload.yml && genny ./workload.yml`. I think it does but just to be sure.